fix: update apple pay (#1539)
saucelabs/sauce-docs: docs/mobile-apps/live-testing/live-mobile-app-testing.md

* Using real private devices with an Apple Pay Sandbox Testing account
* Using real private devices with a real production account and real credit cards

### Requirements
- You need to use [Private devices](#apple-pay-on-real-private-devices)
- [Instrumentation](#disable-instrumentation) needs to be disabled

> - You need to add your Sauce Labs hosted Private device [UDID](#apple-pay-on-real-private-devices) into your own provisioning profile.

Reviewer: ```suggestion
- You need to add your Sauce Labs hosted Private device [UDID](#apple-pay-on-real-private-devices) to your own provisioning profile.
```

> ### Requirements
> - You need to use [Private devices](#apple-pay-on-real-private-devices)
> - [Instrumentation](#disable-instrumentation) needs to be disabled
> - You need to add your Sauce Labs hosted Private device [UDID](#apple-pay-on-real-private-devices) to your own provisioning profile.
> - Devices need to have a physical home button, meaning iPhone SE(2020/2022)/6 series/7 series/8 series. A physical button will ask for confirming the payment with a passcode. 
> 
> :::note
> Devices with a notch (like the iPhone X(S)/11/12/..) will ask for FaceId confirmation. Because this is not enabled the payment will fall back to a different confirmation method. It will use a double press on the power button for the payment approval, but this is not supported.
> :::

Reviewer: ```suggestion
### Requirements
* You need to use [Private devices](#apple-pay-on-real-private-devices).
* [Instrumentation](#disable-instrumentation) needs to be disabled.
* You need to add your Sauce Labs hosted Private device [UDID](#apple-pay-on-real-private-devices) to your own provisioning profile.
* Devices need to have a physical home button (for instance, iPhone SE(2020/2022)/6 series/7 series/8 series). A physical button will require the passcode for payment confirmation. 

:::note
Devices with a notch (like the iPhone X(S)/11/12/..) will ask for FaceId confirmation. As this feature is disabled, the payment will fall back to a different confirmation method. It will use a double press on the power button for the payment approval. However, this method is not supported.
:::
```

To give you an example, Android apps can be installed without any specific signing on whatever real device you want. With Apple you have two options, or you need to add a remote device to your developer certificate and the provisioning profile, so you are allowed to install the app on that specific device. Or you need to use an enterprise certificate where the Apple device that has that certificate installed allows you to install the app. Similarly, when you install an iOS app on a device, we re-sign the app with a Sauce Labs enterprise certificate so you can install your app on all Sauce Labs public/private devices.

:::note
Apple Pay has a limitation that it cannot work with an enterprise certificate. You need to use the developer certificate where the device has been added to the provisioning profile in order to make this work. This can only be done for Sauce Labs private devices on which you have disabled the instrumentation.
:::

### Apple Pay on Real Private Devices

> :::note
> Our real devices are cleaned after every session. This means that you need to configure your Apple Pay Sandbox Testing account, including a passcode and sandbox cards, every time you want to test Apple Pay on an iOS real device.
> :::
> 

Reviewer: ```suggestion
:::note
Our real devices are cleaned after every test session. Therefore, you need to configure your Apple Pay Sandbox Testing account, including a passcode and sandbox cards, every time you want to test Apple Pay on an iOS real device.
:::

```

* Using real private devices with an Apple Pay Sandbox Testing account
* Using real private devices with a real production account and real credit cards

### Requirements
- You need to use [Private devices](#apple-pay-on-real-private-devices)

> - [Resigning](#disable-re-signing) needs to be disabled

Reviewer 2: Resigning is the capability for automated. this is livetesting so lets rename this to Instrumentation so it is matching with the UI otherwise it will be super confusing

3. Prepare the device. Set up the first Sauce Labs private device to use Apple Pay with the Apple sandbox account that was created in step 1.

### Passcode

> Apple Pay requires that you have set a passcode on your phone, and you can’t add cards to your wallet without it. Setting a passcode will be part of [Add the Testing Account](#add-the-testing-account) in the next section.

Reviewer: ```suggestion
One of the Apple Pay requirements is having a set passcode on your phone. Without it, you won't be able to add cards to your wallet. To learn more about setting a passcode, see [Adding the Testing Account](#add-the-testing-account) section.
```
